### ISSUE TITLE
Variables in the Autos window/Locals view are missing with Python3.7.

### ISSUE THREAD

**Issue by linette-zyy** [microsoft/PTVS#6518]
**Describe the bug**
![image](https://user-images.githubusercontent.com/59192491/119102922-24f1b300-ba4d-11eb-9118-049da4b682e8.png)

**Steps to Reproduce**
1. Launch VS and create new python application
2. Add an input('>') or raw_input('>') at the top of your code - depends on version of python
3. Add some more python code after that with breakpoint(s)
4. Start WITHOUT debugging
	a. Console will wait for you to input something
5. Debug > Attach to Process, select the appropriate python.exe (easiest if there's only one, you may want to close interactive windows first) 
6. Type enter in the console
7. Check variables in Autos window.

**Actual behavior**
After press F10 it still just show "..." and cannot see variables.
![image](https://user-images.githubusercontent.com/59192491/119103513-dc86c500-ba4d-11eb-97de-2740e26ac839.png)
**Expected behavior**
You can inspect in the Variables in the Autos window/Locals view.
![image](https://user-images.githubusercontent.com/59192491/119104126-7a7a8f80-ba4e-11eb-98e9-1fe556b3cf2c.png)

**Comment by ijoosong** [comment]
> The part about Autos is #6543
> 
> For Locals, I think it's because you're using Attach to Process, and that still uses the legacy debugger. It should be fixed when we do [microsoft/PTVS-Pylance#152](https://github.com/microsoft/PTVS-Pylance/issues/152).

Can we close this @int19h now that [#152](https://github.com/microsoft/PTVS-Pylance/issues/152) is completed?

**Comment by int19h** [comment]
Yep, https://github.com/microsoft/PTVS/issues/6543 should be the only one still open.
